components/benefit_sponsors/app/controllers/benefit_sponsors/profiles/registrations_controller.rb

```diff
@@ -20,6 +20,8 @@ def new
           set_ie_flash_by_announcement unless is_employer_profile?
         end
 
+        render 'new' unless profile_type.present?
```

Comment:
This is going to be a problem and the fact that this didn't break the build means we haven't spec'ed the scenario we're fixing. So let's write a spec, which should be red because of multiple renders then let's fix the action.